### PR TITLE
(WIP)change v3 to v4 bootstrap

### LINKED ISSUE #938: Upgrade to bootstrap 4
### Please describe the problem (or idea)
We currently use bootstrap 3.7. Bootstrap 4 has some significant improvements/changes. Let's upgrade to v4.

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by ananya**
Fixes #938

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
Is this still a work in progress?

**Comment by Harshithpabbati** [comment]
Send the screenshot and mention publiclab/is-reviewers after you are done :)

**Comment by ananya** [comment]
@HarshKhandeparkar the issue is still in progress. I will let you know once done.

**Comment by HarshKhandeparkar** [comment]
Ok. Np. Feel free to ask any other queries as well.

**Comment by ananya** [comment]
@HarshKhandeparkar i have updated most of the classes i came across. 
while changing i got some bugs. Could you please help me with them.

The load-image toggle is coming open on default. While searching I found that there is "dropdown open" class for the same but it didn't help me. [link](https://stackoverflow.com/questions/24113223/bootstrap-dropdown-menu-defaults-to-open-when-site-loads)
 
Screenshot from 2019-04-05 09-57-54


Initially there was a slider for the module select but it is not coming now.
Screenshot from 2019-04-05 09-57-58


Screenshot from 2019-04-05 09-58-00


Click picture button is coming in two lines.
Screenshot from 2019-04-05 10-15-39

**Comment by Harshithpabbati** [comment]
@ananya please see that all the quick add buttons are in the same line .

**Comment by Harshithpabbati** [comment]
So that we can have the slidebar for smaller screens.

**Comment by ananya** [comment]
@publiclab/is-reviewers I have changed most of the classes i came cross. Please help me in fixing thee bugs I came across.
Screenshot from 2019-04-06 21-56-30
The load image button is not coming in the complete line

Screenshot from 2019-04-06 21-56-35
Screenshot from 2019-04-06 21-56-48
The scroll bar is not coming

Screenshot from 2019-04-06 21-56-50

**Comment by Harshithpabbati** [comment]
@ananya check there's an error in the intermediateHtmlStepUi.js. 
Travis ci failed.

File: examples/lib/intermediateHtmlStepUi.js [hunk 1]
```diff
@@ -108,8 +120,13 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
     $(step.ui.querySelector('.inserDiv .add-step-btn')).prop('disabled', true);
 
     insertStepSelect.append('<option value="none" disabled selected>More modules...</option>');
+<<<<<<< HEAD
```
Comment (Harshithpabbati) [inline examples/lib/intermediateHtmlStepUi.js]: You need to resolve this conflict.

Comment (ananya) [reply examples/lib/intermediateHtmlStepUi.js]: done

**Comment by HarshKhandeparkar** [comment]
This has become very old. Closing for now. Please feel free to rebase and reopen :)

**Pull request closed without merging**